pages/blog/mistake-odyssey.md

```diff
@@ -0,0 +1,62 @@
+
+For a writer or content creator, this nightmare can be especially hurtful, since the ability to write good texts and communicate information does not imply being an expert in computers or in the operation of the software that serves as a platform on which to shape texts.
+
+## Getting out the comfort zone
+
+When we are trained, we learn tools for our profession that we tend to hold on to tightly. Our first jobs are usually done with those initial tools and little by little we discover new ones that open up a world of infinite possibilities. Of course, it is not always easy to let go of the everyday to tackle new challenges with different methods. Homer first learned to recite by heart... at some point he would learn to write... and to read verses.
+
+Sometimes opportunities appear when you least expect them. Thus AsyncAPI appeared in my life and the opportunity to contribute to this project arose. Initially it was an exciting and challenging idea in a totally unknown sector for me, coming from the world of communication. I began to familiarize myself with the concepts and objectives covered by the project and little by little I started to get into it until I came across new tools and working methods that were new to me. Outside my comfort zone, an incomprehensible world was opening up before me. Are there maps for the desert? Who brought me here? Why learn something new if I have already internalized the mechanisms of my previous work?
+
+## Learning from mistakes
+
+AsyncAPI is an open, free and collaborative project so the community needs to work with tools that allow working from transparency and horizontality creating an agile system. We are talking about Git, GitHub and VS Code, three free and open source software tools. 
+
+When it comes to writing, the project relies on plain text, specifically the use of Markdown. It is considered a language that has the purpose of allowing the creation of content in a simple way of writing, and that at all times maintains a readable design. VS Code is the text editor in which we will work with Markdown. And then Git and GitHub conform the system that allows to store the information and easily retrieve it in case of loss, failure or error, as well as to consult the changes and revert them at any time.
+
+Evidently my immersion in these programs was for weeks an odyssey through the vast ocean of ignorance, or what is the same, an odyssey through the purest science of trial and error in which, apparently, there were only errors. Empirical learning is hard.
+
+Fortunately AsyncAPI has a community that works as a great team, in it I found people who help me to dispel my doubts and solve my mistakes. I take this opportunity to thank you from these lines, especially @anbreaker for your patience 😅.
```

Comment:
```suggestion
Fortunately AsyncAPI has a community that works as a great team, in it I found people who help me to dispel my doubts and solve my mistakes. I take this opportunity to thank you from these lines, especially [@anbreaker](https://github.com/anbreaker/) for your patience 😅.
```

This is not automatically converted to a link so I thought it would be useful you link to Antunez's Github profile.

Reply:
Thanks!